e2e/components/dialog/create-from-template-dialog.ts

```diff
@@ -49,7 +49,7 @@ export class CreateFromTemplateDialog extends GenericDialog {
   }
 
   async isValidationMessageDisplayed(): Promise<boolean> {
-    return (await this.validationMessage.isPresent()) && (await this.validationMessage.isDisplayed());
+    return (await this.validationMessage.isPresent()) && (this.validationMessage.isDisplayed());
```

Comment:
I really don't see how this can work.
(boolean && Promise<boolean>) would return true when the first condition is true, the second condition would be completely ignored, won't it?

Reply:
did we already discuss about it?